typescript/hx-adl/src/gen-sql-alter-format.ts

```diff
@@ -0,0 +1,573 @@
+import * as adlast from './adl-gen/sys/adlast';
+import * as adl from "./adl-gen/runtime/adl";
+import { createJsonBinding } from "./adl-gen/runtime/json";
+import { collect, scopedName, scopedNamesEqual, expandTypes, expandNewType, expandTypeAlias, parseAdl, forEachDecl, getAnnotation, decodeTypeExpr, LoadedAdl } from "./util";
+import * as fs from "fs";
+import * as mustache from "mustache";
+import { isEnum, typeExprToStringUnscoped } from './adl-gen/runtime/utils';
+import { Command } from "commander";
+import { snakeCase } from "change-case";
+
+export function configureCli(program: Command) {
+  program
+    .command("sql-alter-format [adlFiles...]")
```

Comment:
@gmhta Do I remove the .command and replace it with .option('--as-alter-statements', 'Generate sql in alter format')? (as suggested here: https://github.com/helix-collective/helix-adl-tools/pull/45#pullrequestreview-598233464)

Reply:
I say yes, replace it with option `--as-alter-statements` (default it to `false` so that existing code doesn't change without opt-in).
It should go back into `gen-sqlschema.ts` to re-use the common part of the code.

Happy to pair with you re-merging it, would be quicker than iterating through code reviews.

Reply:
@paul-thompson-helix I've added an option for a flag back into `gen-sqlschema.ts`. Ready to pair when you have some time today?

Reply:
So it looks like you've left in the file gen-sql-alter-format.ts - it should be merged into gen-sqlschema.ts therefore this file would disappear